### PR TITLE
Fix NApp packaging

### PR DESCRIPTION
**Pull request by lucasgsfelix**
Solving issue related to Napp Packing

### CODE REVIEW

File: kytos/utils/napps.py [hunk 1]
```diff
@@ -466,10 +466,13 @@ def build_napp_package(napp_name):
 
         """
         files = []
-        path = '/'.join(os.path.abspath(__file__).split('/')[:-1])+'/'
+        path = os.getcwd()
+
         for dir_file in os.walk(path):
             files.extend([dir_file[0] + '/' + file for file in dir_file[2]])
 
+        files = list(filter(lambda x: napp_name in x, files))
```
Comment (hdiogenes) [inline kytos/utils/napps.py]: Please add code comments so that we can better understand what are the intentions behind these path transformations.

Comment (lucasgsfelix) [reply kytos/utils/napps.py]: I added the comments, if you have any suggestions I can try to get the code more clear.

File: kytos/utils/napps.py [hunk 2]
```diff
@@ -486,11 +493,11 @@ def build_napp_package(napp_name):
                 if re.search(ignored+"$", filename):
                     files.remove(filename)
                     break
-
         # Create the '.napp' package
         napp_file = tarfile.open(napp_name + '.napp', 'x:xz')
         for local_f in files:
-            napp_file.add(local_f)
+            # remove the path name from the file name
+            napp_file.add(local_f.replace(path+'/', ''))
```
Comment (hdiogenes) [inline kytos/utils/napps.py]: "Add relative paths instead of absolute paths."

File: kytos/utils/napps.py [hunk 3]
```diff
@@ -486,11 +493,11 @@ def build_napp_package(napp_name):
                 if re.search(ignored+"$", filename):
                     files.remove(filename)
                     break
-
         # Create the '.napp' package
         napp_file = tarfile.open(napp_name + '.napp', 'x:xz')
         for local_f in files:
-            napp_file.add(local_f)
+            # Add relative paths instead of absolute paths
+            napp_file.add(local_f.replace(path+'/', ''))
```
Comment (hdiogenes) [inline kytos/utils/napps.py]: Use of `Path` objects and their methods can make this code more readable.
(eg.: [Path.relative_to](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to))

**Pull request merged**